server/models/models.js

```diff
@@ -1,73 +1,40 @@
 var SqlString = require('sqlstring');
-const db = require('../db/index.js');
+const pool = require('../db/index.js');
 
-db.connect();
+pool.connect();
+
+var newId =  200000001;
 
 module.exports = {
 
-  //returns one meal 
-  select: (meal, id) => {
-    return new Promise(function(resolve, reject) {
-      const query = `SELECT * FROM ${meal} WHERE ${SqlString.escape(id)};`
-      db.query(query, function(error, data){
-        if(error) {
-          reject(error);
-        }
-        resolve(data);
-      });
-    });
+  //reads a full menu (many items by restaurant_id);
+  select: async (meal, id) => {
+    var query = `SELECT * FROM ${meal} WHERE restaurant_id = ${SqlString.escape(id)};`;
+    return await pool.query(query);
   },
 
-  //inserts one meal
-  insert: (meal, id, data) => {
-    return new Promise(function(resolve, reject){
-      const query = 
-      `INSERT INTO ${meal} (id, name, description, price, restaurant_id) VALUES (${data.id}, '${data.name}', '${data.description}', ${data.price}, ${data.restaurant_id})`;
-        db.query(query, function(error){
-        if (error){
-          reject(error);
-        }
-
-        var successNotification ='Meal has been added!';
-        resolve(successNotification);
-      });
-    });
+  //inserts one item
+  insert: async (meal, id, data) => {
+    newId += 1; 
```

Comment:
You'll run into issues with keeping track of a newId in your server, because you'll be horizontally scaling and this number won't be shared across all your spinoff servers. You should make your schema in a way where you don't worry about this auto-incrementing number.